### DISCUSSION TITLE
Welcome to riksdagen-corpus Discussions!

### DISCUSSION THREAD

**Discussion by BobBorges** [welfare-state-analytics/riksdagen-corpus#447] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Lauler** [comment]
I work at KBLab at the National Library of Sweden. My goal is to use segmented speeches from `riksdagen-corpus` to align text transcripts against audio recordings (from 1966 and forward). If the segmentation of the text transcripts are of high quality, I will hopefully be able to pinpoint where in an audio/video file that a speech was given. 

I want to build an ASR (audio speech recognition) dataset out of the transcripts and audio. But I will of course share any metadata of when speeches are given with this project and the Riksdag.

**Reply by BobBorges** [reply]
Hi and welcome. Maybe this could be of some use to you: https://github.com/bobborges/elan-asr
A colleague had some success aligning audio to text with European Parliament data, so it is likely feasible here too! Good luck and let us know if/when you have questions.
